README.md

```diff
@@ -716,6 +717,19 @@ __AWESOME__ apps counter: __253__ 🎉
 - [x] [GitHub](https://github.com/Pool-Of-Tears/Myne)
 - [ ] Official page
 
+## Education
```

Comment:
Just change to the [Knowledge](https://github.com/Psyhackological/AAA?tab=readme-ov-file#knowledge) section underneath the Linux Command Library app.

Reply:
Done